src/scripts/wizarduserpage.js

```diff
@@ -36,7 +38,12 @@ define(["loading", "globalize", "dashboardcss", "emby-input", "emby-button", "em
     }
 
     function onSubmit(e) {
-        return submit(this), e.preventDefault(), !1
+        
+        var form = this;
+        return form.querySelector("#txtManualPassword").value != form.querySelector("#txtPasswordConfirm").value ? require(["toast"], function(toast) {
+            toast(Globalize.translate("PasswordMatchError"))
+        }) : submit(this), e.preventDefault(), !1
```

Comment:
This is not that easy to read. I'm guessing you were trying to follow the coding style, but this file has been the victim of `uglify` and it's a long process of undoing it to all those poor JS files. We believe the original owner kept the real source in a private repo.

Anyway, I digress. Suggestion:
```
        var form = this;
        if (form.querySelector("#txtManualPassword").value != form.querySelector("#txtPasswordConfirm").value) {
            require(["toast"], function(toast) {
                toast(Globalize.translate("PasswordMatchError"));
            });
        } else {
            submit(form);
        }
        e.preventDefault();
        return false;
```

Reply:
I addressed your changes.